src/tools/Merger.php

```diff
@@ -57,6 +57,8 @@ public function mergeBill(BillInterface $first, BillInterface $other): BillInter
             $first->getPlan(),
             $charges
         );
+        $bill->setUsageInterval($first->getUsageInterval());
```

Comment:
```suggestion
        $bill->setUsageInterval($first->getUsageInterval()->extend($other->getUsageInterval());
```

Здається, має бути так